corehq/apps/change_feed/producer.py

```diff
@@ -23,7 +23,8 @@ def producer(self):
             api_version=settings.KAFKA_API_VERSION,
             client_id="cchq-producer",
             retries=3,
-            acks=1
+            acks=1,
+            key_serializer=lambda key: key.encode()
```

Comment:
Expecting that this is always a unicode string, but with no special characters. 

@nickpell I think this is fine for python3. I pulled this from the py3 docs and ran the following in py3:

```
>>> blah = u'blah'
>>> type(blah)
<class 'str'>
>>> blah.encode()
b'blah'
```

Reply:
👍